packages/destination-actions/src/destinations/metronome/sendEvent/generated-types.ts

```diff
@@ -0,0 +1,26 @@
+// Generated file. DO NOT MODIFY IT BY HAND.
+
+export interface Payload {
+  /**
+   * The Metronome transaction ID uniquely identifies an event to ensure Metronome only processes each event once.
+   */
+  transaction_id: string
+  /**
+   * The Metronome customer ID or ingest alias this event should be associated with.
+   */
+  customer_id: string
+  /**
+   * The timestamp at which this event occurred.
+   */
+  timestamp: string | number
```

Comment:
this type seems incorrect. We only support an RFC 3999 string right?

Reply:
This is autogenerated by the fields for this action. Specifically it represents the type of the data segment gives us. Our action then needs to turn this into a format our API can accept. That happens here https://github.com/Metronome-Industries/segment-action-destinations/blob/add-metronome-destination/packages/destination-actions/src/destinations/metronome/sendEvent/index.ts#L9

and is tested here

https://github.com/Metronome-Industries/segment-action-destinations/blob/add-metronome-destination/packages/destination-actions/src/destinations/metronome/sendEvent/__tests__/index.test.ts#L55

Reply:
I see serializeEvent converts this. Do you think there might any issues with people passing in non-RFC3999 dates and getting unexpected timestamps?

Reply:
See my other comment.